src/requests/get-l1-content.ts

```diff
@@ -0,0 +1,152 @@
+import { CardContent } from '../interfaces/cardContent';
+
+const getL1Content = (): CardContent[] => {
+    return [
+        {
+            pillCategory: 'DEMO APP',
+            thumbnail: {
+                size: 'medium',
+                url: 'https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/ATF_720x720_17fd9d891f.png',
```

Comment:
if your going to reference `https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/` in a few places I would just store it in a const like 

`const AWS_URL = 'https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/'`